### PR TITLE
data:DevasyaRajguru

### PR DESCRIPTION
**Pull request by devasyarajguru**
## Changes proposed

Added name and social media handles to get my LinkFree account

## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ x] My change requires changes to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Screenshots
Capture

## Note to reviewers

None

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7578"></a>

### CODE REVIEW

**Comment by FahimFBA** [comment]
> ## Changes proposed
> Added name and social media handles to get my LinkFree account
> 
> ## Check List (Check all the applicable boxes)
> * [ x] My code follows the code style of this project.
> * [ x] My change requires changes to the documentation.
> * [ x] I have updated the documentation accordingly.
> * [ x] All new and existing tests passed.
> * [ x] This PR does not contain plagiarized content.
> * [ x] The title of my pull request is a short description of the requested changes.
> 
> ## Screenshots
> Capture
> ## Note to reviewers
> None
> 
> [](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7578)

Make sure not to put any space between `[` and `x`. It would be `[x]` to get a tick mark.

File: data/DevasyaRajguru.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://github.com/devasyarajguru",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/devasya-rajguru-1a38b0211/",
+      "icon": "FaLinkedin"
+    }
+  ]
+
```
Comment (FahimFBA) [inline data/DevasyaRajguru.json:22]: Remove extra new line.

**Review by FahimFBA — CHANGES_REQUESTED** [review]

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please rename your filename to your github username `data/
devasyarajguru.json`

File: data/devasyarajguu.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Devasya Rajguru",
+  "bio": "Lawyer turned Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```
Comment (SaraJaoude) [inline data/devasyarajguu.json:3]: @devasyarajguru we can now merge your PR but are you sure you want the section `Lawyer turned`? This seems to be a copy of the template profile and you may have forgotten to delete it?
```suggestion
  "bio": "Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```

**Comment by FahimFBA** [comment]
> @FahimFBA please check out my code is correct or not?

Resolve the review from @SaraJaoude

**Review by devasyarajguru — COMMENTED** [review]
@SaraJaoude Thanks for mentioning my mistake, I think the problem is solved. Please check it ou!

File: data/devasyarajguu.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Devasya Rajguru",
-  "bio": "Lawyer turned Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
+  "bio": "Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```
Comment (FahimFBA) [inline data/devasyarajguu.json:3]:
```suggestion
  "bio": "Hey, it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```

**Review by FahimFBA — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by devasyarajguru** [comment]
I cannot find my profile in the Linkfree project, Please help me with that! @SaraJaoude

**Comment by devasyarajguru** [comment]
I cannot see my Profile Photo on my LinkFree Account. Please help me with that! @SaraJaoude 
abcd

**Comment by devasyarajguru** [comment]
Hey @SaraJaoude , Please check out my query! Thanks!

**Pull request merged**